ztp/source-crs/optional-extra-manifest/ipsec/enable-ipsec.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: operator.openshift.io/v1
```

Comment:
should these configs be applied at install time or config time? 

extra-manifest indicates install time - at least this is the convention for other files under extra-manifests